src/compiler/reviews.md

```diff
@@ -1,4 +1,159 @@
-# Review policies
+Review Policy
+=============
+
+It is the purpose of code reviews to
+
+- **reduce** the risk of introducing **bugs** and usability and performance **regressions**,
+- keep our code **maintainable**, **readable**, and **documented**, and
+- ensure that changes are made with the big picture and **appropriate context in mind**.
+
+Reviewing accomplishes this by bringing in another set of eyes
+to look at a proposed change from a different perspective,
+which increases the chance of catching mistakes early and
+uncovering potential blind spots in the reasoning behind the change.
+
+
+## Basic Reviewing Requirements
+
+There are a number of requirements that need to be met in order for reviewing to be effective:
+
+- Reviewers must have a sufficient **understanding of the code** under review.
+    > This is important to help spot non-obvious, unintentional side effects of a given change.
+- Pull requests must provide (1) a concise high-level **description of the change** and (2) the **rationale** behind it.
+  For the rationale to be even more useful, authors are encouraged to list potential **points of contention**.
```

Comment:
Besides contention, it would also be good for the author to point the reviewer to anything the author was unsure about (e.g., solutions that didn't seem ideal, compromises that needed to be made, etc.). In other words, what are the "known unknowns" of this PR from the perspective of the author? Perhaps this can be added to the checklist?

Reply:
Done